.travis.yml

```diff
@@ -14,4 +15,5 @@ script: vendor/bin/phpunit
 
 matrix:
   allow_failures:
+    - php: 7.0
```

Comment:
Which both? I only added PHP7 here.

I placed it on allowed failure because of unstable version of PHP, this is just for prepare.

What do you want exactly?